webapp/src/modules/translation/locales/es.json

```
@@ -686,7 +687,8 @@
       "tiara": "Tiara",
       "top_head": "Cabeza",
       "upper_body": "Torso",
-      "skin": "Skin"
+      "skin": "Skin",
+      "hands_wear": "Manos"
```

Following the other named categories, what if we name it `hands`?
```suggestion
      "hands": "Manos"
```